include/JSystem/J3D/J3DPacket.h

```diff
@@ -27,9 +27,27 @@ struct J3DDrawPacket : public J3DPacket {
  * @size{0x40}
  */
 struct J3DMatPacket : public J3DDrawPacket {
-	virtual void entry(J3DDrawBuffer*); // _00
-	virtual void draw();                // _04
-	virtual ~J3DMatPacket();            // _08
+	virtual void entry(J3DDrawBuffer*); // _08
+	virtual void draw();                // _0C
+	virtual ~J3DMatPacket();            // _10
+
+	u32 _0C;                       // _0C
+	u32 _10;                       // _10
+	u8 _14[0xC];                   // _14
+	struct J3DDisplayListObj* _20; // _20
```

Comment:
If possible, declare these before the struct itself

i.e.

```
struct a;
struct b;

struct C {
a* m_inst;
b* m_inst2;
};
```

Reply:
These are from Sodium's branch, and the headers will require further cleanup when that gets sorted out, I've just pulled these in for now. Will kick that can down the road since it'll have to be redone later.